pyaedt/application/Analysis3D.py

```diff
@@ -503,6 +511,10 @@ def export_3d_model(
             List of objects to export. The default is ``None``.
         removed_objects : list, optional
             The default is ``None``.
+        major_version : int, optional
+            File format major version. Default is -1.
+        minor_version : int, optional
+            File format major version. Default is -1.
```

Comment:
```suggestion
            File format for the major version. The default is ``-1``.
        minor_version : int, optional
            File format for the minor version. The default is ``-1``.
```
Are the defaults for both the major or minor versions both ``-1``, or is this a copy/paste error?

Reply:
The default are -1. AEDT interprets it as "latest" version